### DISCUSSION TITLE
Different viewing "modes"

### DISCUSSION THREAD

**Discussion by snaff9** [sumatrapdfreader/sumatrapdf#3337] [Ideas]
Hi,

I think it would be pretty helpful if Sumatra had some different view "modes" e.g. https://github.com/Lunarequest/NightPDF specifically in [this GIF](https://raw.githubusercontent.com/Lunarequest/NightPDF/mistress/docs/nightpdf.gif).

Does something like this already exist?  Apologies if this is a bad question or whatever, this is my first time starting a discussion (in any repo).

**Comment by GitHubRulesOK** [comment]
PDF rendering is convert to screen pixels so you can you can reverse colours to inverted with <kbd>I</kbd> thats not a problem, you can also use Windows NightMode to reduce the more damaging blue colour so the screen looks warmer and pink to reduce eye strain.
You can also shift the color gamut range of images to more or less muted or set the background paper to sepia colour.

All of those are present as settings you can adapt either at starting (command line) or run time advanced options (SumatraPDF-settings.txt)

What you cannot change in SumatraPDF is the way the screen driver hardware shows red green and blue so reversed canvas render un-naturally reverses imbedded images in a PDF thus flipping the colours to negative. Nor can canvas colour change the default interface colours those can be changed by 3rd party software.
